packages/generate-persisted-query-manifest/src/__tests__/cli.test.ts

```diff
@@ -1034,7 +1096,19 @@ test("gathers and reports all errors together", async () => {
       "1:1  error  Operation named "HelloQuery" already defined in: src/hello-query.graphql",
       "src/query.graphql",
       "1:1  error  Anonymous GraphQL operations are not supported. Please name your query.",
-      "✖ 5 errors",
+      "src/syntax-error.graphql",
+      "2:11  error  GraphQLError: Syntax Error: Expected Name, found <EOF>.",
+      "at syntaxError (node_modules/graphql/error/syntaxError.js:15:10)",
```

Comment:
I personally subscribe to the philosophy that when CLI tools report errors that result from bad user input, a stack trace (in the CLI's own code) shouldn't be printed. (Only if the tool is printing out an error that indicates an unexpected bug in the CLI tool is an internal stack trace helpful.) What do you think?

Reply:
That is totally reasonable! My only reasoning is that it made it that much more obvious that there was something horribly wrong with this file, but not sure its a good enough justification to keep it.

Here is what it looks like removed:

```
$ npx generate-persisted-query-manifest
src/bad.graphql
   3:1  error  GraphQLError: Syntax Error: Expected Name, found <EOF>.

src/test.js
  3:11  error  SyntaxError: Missing initializer in const declaration. (3:11)

✖ 2 errors
```

Better?

Reply:
[31c7076](https://github.com/apollographql/apollo-utils/pull/345/commits/31c707608639caf94a137fcc35253e4b4f1ca0c6)

Reply:
I do think that's better — it doesn't seem like understanding the internal details of Babel etc is helpful for our users. (If we find that folks run into problems that do actually involve debugging babel, we can add the trace back in some sort of debug mode?)

Reply:
Yep I like that idea! Let's leave it off for now and can revisit if/when we need it later.